WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -40,12 +43,16 @@ public static void Main(string[] args)
 			{
 				Global = CreateGlobal();
 
+				SingleInstanceChecker = new SingleInstanceChecker(Global.Network);
```

Comment:
Can we possibly move this command before `Global = CreateGlobal();` by extracting config reading from `CreateGlobal()`?

Reply:
Good idea